apps/governance.mento.org/app/components/proposal-list.tsx

```diff
@@ -99,8 +99,16 @@ export const ProposalList = () => {
     veMentoBalance.value,
     proposalThreshold,
   ]);
-  const totalPages = Math.ceil(proposals.length / ITEMS_PER_PAGE);
-  const paginatedProposals = proposals.slice(
+
+  // Sort proposals in descending order by proposalId (latest first)
+  const sortedProposals = useMemo(() => {
+    return [...proposals].sort(
+      (a, b) => Number(b.proposalId) - Number(a.proposalId),
```

Comment:
this is not what i meant. the sorting should be by date (newest first, oldest last)

but the auto-incrementing numbers on the far left of the proposal row should be inverted (beginning with highest number, ending with lowest number)